### ISSUE TITLE
Deepin Get stuck at 68%

### ISSUE THREAD

**Issue by jseyfferdt** [gaining/Resetter#27]
Hi,  thanks for your great work!  I tested it out in the past(made tutorial videos), and you fixed a problem with a password immediately, thank you!

I was trying Deepin out today, and when it is installing packages it gets stuck at 68% at configuring Wireshark.  Again it is not a big problem, a person simply press space and it fix the problem.

But I thought I should just inform you!

**Comment by gaining** [comment]
Hi thanks for letting me know. Could you send me a copy of your log file so that I can troubleshoot this issue further? The log file is located at /var/log/resetter/resetter.log.


Sent from Outlook<http://aka.ms/weboutlook>
________________________________

**Comment by jseyfferdt** [comment]
Sure, I will send it to you tomorrow, my pc is at work.

**Comment by jseyfferdt** [comment]
Here you are!  That's is after I gave it ample of time and then press space.

2017-08-23 03:05:32,484 - __main__ - warningPrompt - WARNING - auto reset chosen
2017-08-23 03:05:32,485 - __main__ - getInstalledList - INFO - getting installed list...
2017-08-23 03:05:32,569 - __main__ - getInstalledList - DEBUG - installed list was generated with 1549 apps installed
2017-08-23 03:05:32,569 - __main__ - getInstalledList - INFO - getting installed list...
2017-08-23 03:05:32,635 - __main__ - getInstalledList - DEBUG - installed list was generated with 1549 apps installed
2017-08-23 03:05:32,635 - __main__ - processManifest - INFO - processing updated manifest...
2017-08-23 03:05:32,637 - __main__ - processManifest - INFO - manifest processing complete
2017-08-23 03:05:32,767 - __main__ - getLocalUserList - INFO - getting local users...
2017-08-23 03:05:32,800 - __main__ - getLocalUserList - INFO - getLocalUserList() completed
2017-08-23 03:05:32,800 - __main__ - findNonDefaultUsers - INFO - getting local users...
2017-08-23 03:05:32,824 - __main__ - findNonDefaultUsers - INFO - getLocalUserList() completed
2017-08-23 03:05:37,164 - ApplyDialog - addUser - INFO - Adding default user...
2017-08-23 03:05:37,164 - ApplyDialog - addUser - INFO - Adding default user...
2017-08-23 03:05:41,619 - ApplyDialog - addUser - INFO - Default user added
2017-08-23 03:05:41,619 - ApplyDialog - addUser - INFO - Default user added
2017-08-23 03:05:46,075 - ApplyDialog - removePackages - INFO - Removing Programs
2017-08-23 03:05:46,075 - ApplyDialog - removePackages - INFO - Removing Programs
2017-08-23 03:05:46,075 - ApplyDialog - removePackages - INFO - Keep Count before commit: 52
2017-08-23 03:05:46,075 - ApplyDialog - removePackages - INFO - Keep Count before commit: 52
2017-08-23 03:05:46,076 - ApplyDialog - removePackages - INFO - Delete Count before commit: 29
2017-08-23 03:05:46,076 - ApplyDialog - removePackages - INFO - Delete Count before commit: 29
2017-08-23 03:05:46,076 - ApplyDialog - removePackages - INFO - Broken Count before commit: 0
2017-08-23 03:05:46,076 - ApplyDialog - removePackages - INFO - Broken Count before commit: 0
2017-08-23 03:06:44,823 - ApplyDialog - removePackages - INFO - Broken Count after commit: 0
2017-08-23 03:06:44,823 - ApplyDialog - removePackages - INFO - Broken Count after commit: 0
2017-08-23 03:06:44,828 - ApplyDialog - fixBroken - INFO - Cleaning up...
2017-08-23 03:06:44,828 - ApplyDialog - fixBroken - INFO - Cleaning up...
2017-08-23 03:06:50,102 - ApplyDialog - onFinished - DEBUG - Cleanup finished with exit code: 0 and exit_status 0.
2017-08-23 03:06:50,102 - ApplyDialog - onFinished - DEBUG - Cleanup finished with exit code: 0 and exit_status 0.
2017-08-23 03:06:50,103 - ApplyDialog - installPackages - INFO - Starting installations...
2017-08-23 03:06:50,103 - ApplyDialog - installPackages - INFO - Starting installations...
2017-08-23 03:06:58,112 - InstallMissingDialog - installPackages - INFO - treating Packages
2017-08-23 03:06:58,112 - InstallMissingDialog - installPackages - INFO - treating Packages
2017-08-23 03:15:24,397 - __main__ - warningPrompt - WARNING - auto reset chosen
2017-08-23 03:15:24,401 - __main__ - getInstalledList - INFO - getting installed list...
2017-08-23 03:15:24,505 - __main__ - getInstalledList - DEBUG - installed list was generated with 1630 apps installed
2017-08-23 03:15:24,506 - __main__ - getInstalledList - INFO - getting installed list...
2017-08-23 03:15:24,599 - __main__ - getInstalledList - DEBUG - installed list was generated with 1630 apps installed
2017-08-23 03:15:24,600 - __main__ - processManifest - INFO - processing updated manifest...
2017-08-23 03:15:24,603 - __main__ - processManifest - INFO - manifest processing complete
2017-08-23 03:15:24,622 - __main__ - getLocalUserList - INFO - getting local users...
2017-08-23 03:15:24,659 - __main__ - getLocalUserList - INFO - getLocalUserList() completed
2017-08-23 03:15:24,659 - __main__ - findNonDefaultUsers - INFO - getting local users...
2017-08-23 03:15:24,678 - __main__ - findNonDefaultUsers - INFO - getLocalUserList() completed
2017-08-23 03:15:29,748 - InstallMissingDialog - cancel - WARNING - Progress thread was cancelled
2017-08-23 03:15:29,748 - InstallMissingDialog - cancel - WARNING - Progress thread was cancelled
2017-08-23 03:15:29,758 - ApplyDialog - removeUsers - INFO - Starting user removal
2017-08-23 03:15:29,758 - ApplyDialog - removeUsers - INFO - Starting user removal

**Comment by gaining** [comment]
Alright, try running it from the terminal and when you get stuck again try pasting the terminal output (text generated on the terminal) on here. I don't think the problem was caught on the log file.

**Comment by gaining** [comment]
I think I may have figured out what's causing the issue. I will push a fix later today.

**Comment by jseyfferdt** [comment]
Hi, I will have a look right now!

**Comment by jseyfferdt** [comment]
Hi, it still get stuck at the same place

**Comment by gaining** [comment]
I've updated resetter, please try the latest version: https://github.com/gaining/Resetter/releases/tag/v1.0.3-stable and tell me if you were able to fix the issue.

**Comment by jseyfferdt** [comment]
Hi, I still get stuck at this window, and here is the log also, sorry for the bit odd shot the screenshot got strange

2017-08-25 02:36:35,830 - __main__ - warningPrompt - WARNING - auto reset chosen
2017-08-25 02:36:35,831 - __main__ - getInstalledList - INFO - getting installed list...
2017-08-25 02:36:35,916 - __main__ - getInstalledList - DEBUG - installed list was generated with 1549 apps installed
2017-08-25 02:36:35,916 - __main__ - getInstalledList - INFO - getting installed list...
2017-08-25 02:36:35,985 - __main__ - getInstalledList - DEBUG - installed list was generated with 1549 apps installed
2017-08-25 02:36:35,985 - __main__ - processManifest - INFO - processing updated manifest...
2017-08-25 02:36:35,988 - __main__ - processManifest - INFO - manifest processing complete
2017-08-25 02:36:36,017 - __main__ - getLocalUserList - INFO - getting local users...
2017-08-25 02:36:36,043 - __main__ - getLocalUserList - INFO - getLocalUserList() completed
2017-08-25 02:36:36,043 - __main__ - findNonDefaultUsers - INFO - getting local users...
2017-08-25 02:36:36,054 - __main__ - findNonDefaultUsers - INFO - getLocalUserList() completed
2017-08-25 02:36:40,666 - ApplyDialog - addUser - INFO - Adding default user...
2017-08-25 02:36:40,666 - ApplyDialog - addUser - INFO - Adding default user...
2017-08-25 02:36:41,545 - ApplyDialog - addUser - INFO - Default user added
2017-08-25 02:36:41,545 - ApplyDialog - addUser - INFO - Default user added
2017-08-25 02:36:46,348 - ApplyDialog - removePackages - INFO - Removing Programs
2017-08-25 02:36:46,348 - ApplyDialog - removePackages - INFO - Removing Programs
2017-08-25 02:36:46,352 - ApplyDialog - removePackages - INFO - Keep Count before commit: 52
2017-08-25 02:36:46,352 - ApplyDialog - removePackages - INFO - Keep Count before commit: 52
2017-08-25 02:36:46,352 - ApplyDialog - removePackages - INFO - Delete Count before commit: 29
2017-08-25 02:36:46,352 - ApplyDialog - removePackages - INFO - Delete Count before commit: 29
2017-08-25 02:36:46,352 - ApplyDialog - removePackages - INFO - Broken Count before commit: 0
2017-08-25 02:36:46,352 - ApplyDialog - removePackages - INFO - Broken Count before commit: 0
2017-08-25 02:37:03,060 - ApplyDialog - removePackages - INFO - Broken Count after commit: 0
2017-08-25 02:37:03,060 - ApplyDialog - removePackages - INFO - Broken Count after commit: 0
2017-08-25 02:37:03,066 - ApplyDialog - fixBroken - INFO - Cleaning up...
2017-08-25 02:37:03,066 - ApplyDialog - fixBroken - INFO - Cleaning up...
2017-08-25 02:37:09,348 - ApplyDialog - onFinished - DEBUG - Cleanup finished with exit code: 0 and exit_status 0.
2017-08-25 02:37:09,348 - ApplyDialog - onFinished - DEBUG - Cleanup finished with exit code: 0 and exit_status 0.
2017-08-25 02:37:09,349 - ApplyDialog - installPackages - INFO - Starting installations...
2017-08-25 02:37:09,349 - ApplyDialog - installPackages - INFO - Starting installations...
2017-08-25 02:37:18,319 - InstallMissingDialog - installPackages - INFO - treating Packages
2017-08-25 02:37:18,319 - InstallMissingDialog - installPackages - INFO - treating Packages
2017-08-25 02:45:03,305 - InstallMissingDialog - cancel - WARNING - Progress thread was cancelled
2017-08-25 02:45:03,305 - InstallMissingDialog - cancel - WARNING - Progress thread was cancelled
2017-08-25 02:45:03,313 - ApplyDialog - removeUsers - INFO - Starting user removal
2017-08-25 02:45:03,313 - ApplyDialog - removeUsers - INFO - Starting user removal
2017-08-25 02:45:05,443 - ApplyDialog - showUserInfo - INFO - Credential message info shown
2017-08-25 02:45:05,443 - ApplyDialog - showUserInfo - INFO - Credential message info shown
2017-08-25 02:45:07,392 - ApplyDialog - rebootMessage - INFO - reboot was delayed.
2017-08-25 02:45:07,392 - ApplyDialog - rebootMessage - INFO - reboot was delayed.

![deepinscreenshot_select-area_20170825024457](https://user-images.githubusercontent.com/26792699/29702574-e1a8d46c-893f-11e7-9642-cb99684abf00.png)

**Comment by gaining** [comment]
ok remove resetter with sudo apt remove resetter, then install the latest one.

**Comment by jseyfferdt** [comment]
i will do it for you over the weekend, sorry for bugging you!

**Comment by gaining** [comment]
No problem. I installed wireshark using resetter in deepin and was unable reproducing the issue. Have you tried installing without?

**Comment by jseyfferdt** [comment]
hi, how do I choose not to install it, when running in the terminal, it locks out when trying to install packages says an app like synaptic is open, but resetter and the terminal is the only apps that are running.  but once i got through and like a video appeared in the terminal of wireshark where you have to move with tab and arrow buttons,  how can I just reset without installing wireshark?  I tried ot find that option.  is it because I run it in virtualbox for testing?

**Comment by gaining** [comment]
Ok I see what happened. What you basically saw was a debconf prompt asking you if non superusers should be able to capture packets. Now, I have already implemented a way for resetter to show the debconf prompt and it should look like this 
![screenshot from 2017-08-26 14-28-32](https://user-images.githubusercontent.com/7690230/29744070-f50ff2b6-8a6a-11e7-9d5e-23ad11fdbeca.png) or this: 
![screenshot from 2017-08-26 14-27-21](https://user-images.githubusercontent.com/7690230/29744073-fe5bd498-8a6a-11e7-83a2-cd8e8806aa88.png)
Now I tried it earlier in deepin and got the same prompt but I'm not sure why it isn't showing up for you.

Your solution is simple. Just install the wireshark package from the terminal without resetter and accept the prompt.  Basically, `sudo apt install wireshark` and accept the debconf prompt
Deepin included the package wireshark-common by default. If you want skip installing this package on a reset, use the custom reset option and deselect wireshark-common on the MIPAS to reinstall.
